File: etc/too_simple.py
#!/usr/bin/env python

def main():
    sumval = 0.0
    flip = -1.0
    for i in range(1, 10000001):
        flip = -flip
        sumval += flip / ((i << 1) - 1)
    print(f"{sumval * 4.0:.16f}")

if __name__ == "__main__":
    #import dis
    #dis.dis(main)
    main()

File: etc/too_simple.lua
#!/usr/bin/env lua

function main()
    local sum = 0.0
    local flip = -1.0
    for i = 1, 100000000 do
        flip = -flip
        --sum = sum + flip / (2 * i - 1)
        sum = sum + flip / ((i << 1) - 1)
    end
    print(string.format("%.16f", sum * 4.0))
end

main()
